python_modules/libraries/dagster-dg/dagster_dg/cli/plus/deploy.py

```diff
@@ -98,6 +98,13 @@ def _get_deployment(input_deployment: Optional[str], plus_config: DagsterPlusCli
 )
 @click.option("--git-url", "git_url")
 @click.option("--commit-hash", "commit_hash")
+@click.option(
+    "--location-name",
+    "location_names",
+    help="Name of the code location to set the build output for. Defaults to the current project's code location, or every project when run in a workspace.",
```

Comment:
I do find bridging the terms `project` and `code location` a bit clunky here, not much we can do I guess.

> Defaults to the current project's code location, or every project when run in a workspace.

As verbose as this is, should this be "or every project's code location, when run in a workspace"?